### PR TITLE
General Housekeeping of readme and admin ranks.

### PR DESCRIPTION
**Pull request by Birdtalon**
- Starts curating the README.md to better reflect our new codebase.
- Removes a bunch of admin ranks - we don't need them at this stage.
- Renames the DM environment, alters the comment in the new planetfall.dm file to reflect the new filename
- Adjusts Travis.yml to point at the new planetfall.dme file
- Adjusts Issue and Pull request templates to remove /tg/station related content.
- Adjusts CONTRIBUTING.md to better reflect our codebase.

### CODE REVIEW

**Comment by SamHPurp** [comment]
Might be worth including updating the enviroment name etc.

**Comment by SamHPurp** [comment]
Lets hope this doesn't hillariously break everything.

File: planetfall.dme [hunk 1]
```diff
@@ -1,2176 +1,2176 @@
...
-#include "code\modules\clothing\shoes\bananashoes.dm"
-#include "code\modules\clothing\shoes\colour.dm"
-#include "code\modules\clothing\shoes\magboots.dm"
-#include "code\modules\clothing\shoes\miscellaneous.dm"
-#include "code\modules\clothing\spacesuits\chronosuit.dm"
-#include "code\modules\clothing\spacesuits\flightsuit.dm"
-#include "code\modules\clothing\spacesuits\hardsuit.dm"
-#include "code\modules\clothing\spacesuits\miscellaneous.dm"
-#include "code\modules\clothing\spacesuits\plasmamen.dm"
-#include "code\modules\clothing\spacesuits\syndi
```
Comment (SamHPurp) [inline planetfall.dme]: `planetfall`

File: README.md [hunk 2]
```diff
@@ -1,14 +1,6 @@
...
-
-[![Build Status](https://travis-ci.org/tgstation/tgstation.png)](https://travis-ci.org/tgstation/tgstation) [![Krihelimeter](http://www.krihelinator.xyz/badge/tgstation/tgstation)](http://www.krihelinator.xyz)  
-[![Percentage of issues still open](http://isitmaintained.com/badge/open/tgstation/tgstation.svg)](http://isitmaintained.com/project/tgstation/tgstation "Percentage of issues still open") [![Average time to resolve an issue](http://isitmaintained.com/badge/resolution/tgstation/tgstation.svg)](http://isitmaintained.com/project/tgstation/tgstation "Average time to resolve an issue") ![Coverage](https://s3.amazonaws.com/assets.coveralls.io/badges/coveralls_0.svg)  
-[![forthebadge](http://forthebadge.com/images/badges/built-with-resentment.svg)](http://forthebadge.com)
-
-**Website:** http://www.tgstation13.org <BR>
-**Code:** https://github.com/tgstation/tgstation <BR>
-**Wiki** http://tgstation13.org/wiki/Main_Page <BR>
-**IRC:** irc://irc.rizon.net/coderbus or if you dont have an IRC client, you can click  [here](https://kiwiirc.com/client/irc.rizon.net:6667/?&theme=cli#coderbus).<BR>
+## Planetfall codebase orginally forked from /tg/station.
```
Comment (SamHPurp) [inline README.md]: `orginally` -> `originally`

File: planetfall.dme (hunk 1 above)
Comment (SamHPurp) [inline planetfall.dme]: Ignore the example it shows.... its in the first sentence or two of the files (open in `notepad++`) is probably easiest.

File: README.md (hunk 2 above)
Comment (Birdtalon) [inline README.md]: Derp!!

**Comment by Birdtalon** [comment]
I am assuming Travis won't pass on this because it's checking for a file which has been renamed.

**Comment by FlattestGuitar** [comment]
https://github.com/Planetfall/PlanetfallSS13/blob/master/tools/travis/build_byond.sh#L23

o7

**Comment by Birdtalon** [comment]
Thanks @FlattestGuitar I can't see anything more to change at this point.

**Pull request merged**